src/ProfilesPage/DeletedUser.js

```diff
@@ -0,0 +1,16 @@
+import React, {Component} from 'react';
+
+class DeletedUser extends Component {
```

Comment:
nit: since there are no lifecycle methods can write component as:
```
const DeletedUser = () => (
    <div id="message">
   .....
  </div>
)
```

Reply:
done